opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -88,9 +88,9 @@ class LogLimits:
     UNSET = -1
 
     def __init__(
-        self,
-        max_attributes: Optional[int] = None,
-        max_attribute_length: Optional[int] = None,
+            self,
```

Comment:
Please don't add these indentations.

Reply:
Thanks for the reminder, it has been restored.